enarx-keep/src/main.rs

```diff
@@ -4,9 +4,14 @@
 
 #![deny(clippy::all)]
 #![deny(missing_docs)]
+#![feature(asm)]
 
 mod backend;
 mod binary;
+mod sallyport;
+
+// workaround for sallyport tests, until we have internal crates
+pub use sallyport::Request;
```

Comment:
Why not `pub mod sallyport`?

Reply:
doesn't make a difference. The macro in the tests wants `$crate::Request`